### PR TITLE
Feature/create header

### PR DESCRIPTION
**Pull request by anastasiya-nd**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import HeaderApp from './components/Header/HeaderApp';
```
Comment (ok2ju) [inline src/App.jsx:2]: Why `HeaderApp` and not just `Header`?

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+const App = () => (
+  <div>
```
Comment (ok2ju) [inline src/App.jsx:5]: Do you need this empty div?

File: src/assets/img/Logo.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<svg width="204" height="32" viewBox="0 0 204 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ok2ju) [inline src/assets/img/Logo.svg:1]: Please, create an react component. I mean create a folder called `Icons` inside of `components` folder and then create a component called `Logo.jsx` and render svg inside

File: src/components/Header/HeaderApp.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+import logo from '../../assets/img/Logo.svg'
+import Navbar from './Navbar/Navbar';
+
+const Header = styled.header`
```
Comment (ok2ju) [inline src/components/Header/HeaderApp.jsx:6]: Please, move styled components into `styles.js` file

File: webpack.config.js [hunk 5]
```diff
@@ -5,17 +5,28 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js$/,
+        test: /\.jsx$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
       },
+      {
+        test: /\.svg$/,
```
Comment (ok2ju) [inline webpack.config.js:13]: You don't need this loader. Create a react components for icons

File: src/components/Header/Navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Nav = styled.nav`
+  ul {
```
Comment (ok2ju) [inline src/components/Header/Navbar/Navbar.jsx:6]: Please, create a separate styled components for each html node

**Review by ok2ju — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (anastasiya-nd) [reply src/App.jsx:2]: Because I use name Header for styles. I'm going to change it

File: src/App.jsx (hunk 2 above)
Comment (anastasiya-nd) [reply src/App.jsx:5]: div is not empty. It has component Header. And this div will have other code

File: webpack.config.js (hunk 5 above)
Comment (anastasiya-nd) [reply webpack.config.js]: I have to create one component, that will receive src and alt  in props. Yes?

File: src/assets/img/Logo.svg (hunk 3 above)
Comment (anastasiya-nd) [reply src/assets/img/Logo.svg:1]: I have done it, but I'm not sure that it's right

File: src/App.jsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Header from './components/Header/Header';
```
Comment (ok2ju) [inline src/App.jsx:2]: I don't like `/Header/Header` in the import path. Please, create `index.js` file in every component folder and export root component. So, after that you will import component using `'./components/Header';` path

**Review by ok2ju — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 5 above)
Comment (ok2ju) [reply webpack.config.js]: Nope, just a react component with svg in render. Thats it.

File: src/components/Header/Header.jsx [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Logo from '../Icons/Logo';
+import Navbar from './Navbar/Navbar';
+import HeaderItem from './styles';
+
+
```
[the remark this answers is not in the document]
Comment (anastasiya-nd) [reply src/components/Header/Header.jsx:6]: No

**Pull request merged**